src/packages/@ncigdc/components/Modals/FirstTimeModal.js

```diff
@@ -29,8 +29,9 @@ const FirstTimeModal = () => (
         GDC Data Release Notes
       </ExternalLink>
       {" "}
-      prior to accessing this web site as the Release Notes provide details about data updates, known issues and workarounds.
+      prior to accessing this web site as the Release Notes provide details about data updates, and known issues and workarounds.
```

Comment:
oh this came back? I remember asking you to remove that `and`.. did someone request this to come back?